src/Diagnostics.ModelsAndUtils/Attributes/Definition.cs

```diff
@@ -92,5 +92,22 @@ public bool Equals(Definition other)
         {
             return Id == other.Id;
         }
+
+        public Definition()
+        {
+        }
+
+        public Definition(Definition definition)
+        {
+            Id = definition.Id;
+            Name = definition.Name;
+            Description = definition.Description;
```

Comment:
Random thought : instead of copying each element, maybe we serialize the object to a string or memory stream and then create a new object by deserializing that.

I am just wondering that if we add a new member to this class, we might forget to update here

Reply:
I agree Shekhar, good suggestion!